content/cockpit-129.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 128 and 129
+author: Dominik Perpeet
+date: 2016-01-19 09:00
+tags: cockpit, linux
+slug: cockpit-129
+category: release
+summary: Cockpit supports Rebases and Remotes in OSTree.
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release
+regularly. Here are the release notes from version 128 and 129.
+
+### Manage remotes and rebase in OSTree
+
+When working with [OSTrees](http://www.projectatomic.io/docs/os-updates/) there will
+often be multiple branches to choose from. Thanks to Peter's work, Cockpit can now switch
+between branches and view and activate OSTrees from those branches. Also, Cockpit supports
+managing multiple remotes and viewing their branches.
+Take a look at the video below for a demo.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dF8M3xq9FcI" frameborder="0" allowfullscreen></iframe>
```

Comment:
No need to record this again. But in general we would want to use a real world use case. We would want to avoid ```zremote``` and actually show rebasing from RHEL to Fedora or Fedora Beta -> Fedora 25 ... etc. 

 It may have been hard in this scenario ... but worth keeping in mind.